server/chroma/src/server.rs

```diff
@@ -0,0 +1,112 @@
+use std::time::Duration;
+
+use actix_cors::Cors;
+use actix_web::{web, App, HttpServer};
+use anyhow::{anyhow, bail, Result};
+use cabbage::KoalaApi;
+use noiseless_tracing_actix_web::NoiselessRootSpanBuilder;
+use tracing::{info, warn};
+use tracing_actix_web::TracingLogger;
+
+use dal::database::Database;
+use dal::storage_engine::{S3Config, Storage};
+
+use crate::config::Config;
+use crate::routes;
+use crate::routes::appdata::{AlbumIdCache, AppData, Ratelimits, SessionIdCache, WebData};
+use crate::routes::routable::Routable;
+
+/// Run the chroma server and will block until the server is stopped or crashes
+///
+/// # Errors
+///
+/// If the configuration is invalid
```

Comment:
Shouldn't this be a list-like? I'm not sure, but I think lines should start with a `-`, or at least end with a period `.`